src/main/groovy/io/jmix/build/JmixBuildPlugin.groovy

```diff
@@ -265,4 +266,16 @@ class JmixBuildPlugin implements Plugin<Project> {
     private Set<Project> getJavaSubProjects(Project rootProject) {
         rootProject.subprojects.findAll { subproject -> subproject.plugins.hasPlugin(JavaPlugin) }
     }
+
+    private void setupDependencyManagement(Project project) {
+        if (!project.name.startsWith('sample')) {
+            project.with {
+                dependencies {
+                    api platform("org.springframework.boot:spring-boot-starter-parent:2.5.2")
```

Comment:
Why not `org.springframework.boot:spring-boot-dependencies:2.5.2` which is used in BOM's build.gradle?